<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pozadina.qrc">
    <normaloff>:/pozadina.png</normaloff>:/pozadina.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_32">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>340</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_33">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>185</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>185</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>30</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : pink;
background-color: rgb(0, 0, 0) 0.5;
font: 40px;</string>
    </property>
    <property name="text">
     <string>X je na potezu</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player1Wins">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>210</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);
color: white;
border-radius: 20px;
border-color: red;
font: 18px;
</string>
    </property>
    <property name="text">
     <string>  Igrač1 pobeda: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player2Wins">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>210</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);
color: white;
font: 18px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>  Igrač2 pobeda: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="DrawGames">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>210</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);
color: white;
font: 18px;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>  Draws: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
color: black;</string>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="ind2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>245</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #c0cbff;
color: white;
border-radius: 20px;
font: 18px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ind1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>145</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: pink;
color: white;
border-radius: 20px;
font: 18px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>800</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pozadina.qrc">:/pozadina.png</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButton_31</zorder>
   <zorder>pushButton_32</zorder>
   <zorder>pushButton_33</zorder>
   <zorder>pushButton_21</zorder>
   <zorder>pushButton_22</zorder>
   <zorder>pushButton_23</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>label</zorder>
   <zorder>Player1Wins</zorder>
   <zorder>Player2Wins</zorder>
   <zorder>DrawGames</zorder>
   <zorder>Restart</zorder>
   <zorder>ind2</zorder>
   <zorder>ind1</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pozadina.qrc"/>
 </resources>
 <connections/>
</ui>
